packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -330,7 +328,7 @@ const KMS = WebexPlugin.extend({
           .then(() => {
             /* istanbul ignore else */
             if (process.env.NODE_ENV !== 'production') {
-              this.logger.info('kms: request payload', util.inspect(omit(JSON.parse(JSON.stringify(req)), 'wrapped'), {depth: null}));
+              this.logger.info('kms: request payload');
```

Comment:
i think if we're not printing any details about the payload, we probably shouldn't print anything. Can we anonymize the logs and still keep relevant details?